Resources/Textures/Structures/Holo/security.rsi/meta.json

```diff
@@ -5,12 +5,16 @@
     "y": 32
   },
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/f4017da82ae4de7bfaf8ebdbba33d0de81c15587",
+    "copyright": "Created by Kit0vras#5869",
     "states": [
         {
             "name": "icon",
             "delays": [
                 [
+                    0.3,
+                    0.3,
+                    0.3,
+                    0.3,
                     0.3,
                     0.3,
                     0.3,
```

Comment:
Make the delays a lil shorter so the animation goes quicker, right now it looks laggy

Reply:
I will add more frames as shorter delays didn't solve the problem.